src/app/services/utils.service.ts

```diff
@@ -178,23 +170,45 @@ export class UtilsService {
         }).format(date);
 
       default:
-        return new Intl.DateTimeFormat('en-GB', {
-          hour12: true,
-          hour: 'numeric',
-          minute: 'numeric',
-          month: 'short',
-          day: 'numeric',
-          year: 'numeric'
-        }).format(date);
+      return this.dateFormatter(date) + ' ' + new Intl.DateTimeFormat('en-GB', {
```

Comment:
Do not repeat it, 
`const formattedTime = new Intl.DateTimeFormat('en-GB', ...`
and use it in both `case 'time'` and `default`